backend/src/ds/attachments.rs

```diff
@@ -0,0 +1,198 @@
+// SPDX-FileCopyrightText: 2025 Phoenix R&D GmbH <hello@phnx.im>
+//
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+use aws_sdk_s3::{
+    config::http,
+    error::{BuildError, SdkError},
+    operation::{get_object, put_object},
+    presigning::{PresigningConfig, PresigningConfigError},
+};
+use chrono::{DateTime, Duration, Utc};
+use displaydoc::Display;
+use phnxcommon::{identifiers::AttachmentId, time::ExpirationData};
+use phnxprotos::delivery_service::v1::{
+    GetAttachmentUrlResponse, HeaderEntry, ProvisionAttachmentPayload, ProvisionAttachmentResponse,
+};
+use tonic::{Response, Status};
+use tracing::error;
+use uuid::Uuid;
+
+use super::Ds;
+
+impl Ds {
+    pub(super) async fn provision_attachment(
+        &self,
+        _payload: ProvisionAttachmentPayload,
+    ) -> Result<Response<ProvisionAttachmentResponse>, ProvisionAttachmentError> {
+        let client = self
+            .storage
+            .as_ref()
+            .map(|s| s.client())
+            .ok_or(ProvisionAttachmentError::NoStorageConfigured)?;
+
+        let attachment_id = Uuid::new_v4();
+
+        let expiration = ExpirationData::now(Duration::minutes(5));
```

Comment:
Can you move this to a constant or a hard-coded configuration?

Reply:
Done in 9eb10a2d